lib/membrane_dashboard/dagre/g6_marshaller.ex

```diff
@@ -156,7 +156,7 @@ defmodule Membrane.Dashboard.Dagre.G6Marshaller do
     [label | parents] = path |> Enum.reverse()
 
     parent_id =
-      if length(parents) == 0 do
+      if parents == [] do
```

Comment:
I think `case` suits better here

Reply:
I think that in this case `if` is pretty much equivalent.

Reply:
Maybe it is a matter of opinion. So it can stay if you want